apps/full-site-editing/full-site-editing-plugin/editor-plans-grid/src/plans-modal/index.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Modal } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import PlansGrid, { Props as PlansGridProps } from '@automattic/plans-grid';
+import './styles.scss';
+
+interface Props extends Omit< PlansGridProps, 'header' > {
+	isOpen: boolean;
+	onClose: () => void;
+}
+
+const PlansModal: React.FunctionComponent< Props > = ( { isOpen, onClose, ...props } ) => {
+	if ( ! isOpen ) {
+		return null;
+	}
+
+	const header = (
+		<div>
+			<h1 className="wp-brand-font">{ __( 'Choose a plan', 'full-site-editing' ) }</h1>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Choose a plan' )` *ES Score: 7*